src/rpc/blockchain.cpp

```diff
@@ -390,6 +397,8 @@ UniValue getblock(const UniValue& params, bool fHelp)
             "  \"bits\" : \"1d00ffff\", (string) The bits\n"
             "  \"difficulty\" : x.xxx,  (numeric) The difficulty\n"
             "  \"chainwork\" : \"xxxx\",  (string) Expected number of hashes required to produce the chain up to this block (in hex)\n"
+            "  \"sizelimit\" : n,       (numeric) BIP100 block size limit as of this block\n"
+            "  \"sizelimitvote\" : n,   (numeric) This block's sizelimit vote\n"
```

Comment:
sizelimit -> size limit.
Also, WRT The comment: "BIP100 block size limit as of this block".  This limit is applied to the **next** block, right?  I think that this is the important thing to communicate.  Maybe:
"Size limit of next block as calculated per BIP100."